### PR TITLE
Add MAINTENANCE_MODE setting

### PR DESCRIPTION
**Pull request by alexmuller**
This overrides all the urlpatterns to serve a 503 error for every page with a simple template.

I think we should probably just add the different translations in a static way in this template.

### CODE REVIEW

File: gem/settings/base.py [hunk 1]
```diff
@@ -33,6 +33,8 @@
 DEBUG = True
 ENV = 'dev'
 
+MAINTENANCE_MODE = environ.get('MAINTENANCE_MODE', '') == 'true'
```
Comment (codiebeulaine) [inline gem/settings/base.py]: what does the == true but do?

**Comment by codiebeulaine** [comment]
@alexmuller can we write a test for this?

**Comment by alexmuller** [comment]
@codiebeulaine I've tried to write a test for this but it's not possible to override the `MAINTENANCE_MODE` setting in the test because the urlconf is loaded when the app starts. I also tried to write a test for the view class directly but haven't figured out how to get that working yet.

**Comment by codiebeulaine** [comment]
👍

**Pull request merged**